### ISSUE TITLE
Multiple metrics

### ISSUE THREAD

**Issue by Unitech** [keymetrics/pmx#94]
https://stackoverflow.com/questions/44896885/pm2-metric-for-ram-usage-multiple-metrics-in-one-pmx-metric

**Comment by alavit-d** [comment]
Hey,

`Probe.metric(opts1, opts2)` will only evaluate the first argument `opts1`, so you cannot put several metrics in one declaration.
Instead create an object and fill it probe by probe.

There is currently no way to access the CPU or Mem metrics from inside your app, the values are reported by PM2. So unless you calculate them in your code, you will not be able to report `cpu_usage` or `memory_usage` values as metrics.

**Comment by Unitech** [comment]
@alavit-d Yes we cannot put multiple metrics, that's why we are here, to maintain and make pmx better right..

Re opening issue
